client/galaxy/scripts/components/WaitButton.vue

```diff
@@ -0,0 +1,68 @@
+/** Button variant with waiting and progress tracking. */
+<template>
+    <button
+        class="ui-button-default"
+        v-bind:class="[waiting ? 'disabled' : '', variantClass]"
```

Comment:
I wonder if we should avoid this one-line if statements in the template and consistently use computed values instead. I personally lean towards computed values going forward.

Reply:
I also think computed vs ternary has a clear winner

Reply:
Will do, thanks!

Reply:
Rebased and included in https://github.com/galaxyproject/galaxy/pull/9138 the oldest/smallest PR that contained this. Will rebase these other PRs on top of that - but they are all going to lose their selenium testing :(.